xga/utils.py

```diff
@@ -647,11 +664,47 @@ def check_telescope_choices(telescope: Union[str, List[str]]) -> List[str]:
 else:
     eSASS_AVAIL = True
 
+# Then, we check to see what version of CIAO (if any) is installed - for the Chandra mission
+# Here we check to see whether CIAO is installed (along with all the necessary paths)
+CIAO_VERSION = None
+# This checks for an installation of Ciao
+CIAO_AVAIL = False
+
+ciao_out, ciao_err = Popen("ciaover -v", stdout=PIPE, stderr=PIPE, shell=True).communicate()
+# Just turn those pesky byte outputs into strings
+ciao_out = ciao_out.decode("UTF-8")
+ciao_err = ciao_err.decode("UTF-8")
+
+if "ciaover: command not found" in ciao_err:
+        warn("CIAO cannot be identified on your system, and Chandra data cannot be processed.")
+else:
+    # The ciaover output is over a series of lines, with different info on each - this is a little bit of a hard
+    #  code cheesy method to do this, but we'll split them on lines and selected the 2nd line to get
+    #  the ciao version
+    split_out = [en.strip(' ') for en in ciao_out.split('\n')]
+    # Strip the CIAO version out of the ciaover output
+    CIAO_VERSION = split_out[1].split(':')[-1].split('CIAO')[-1].strip(' ').split(' ')[0]
+    CIAO_AVAIL = True
+
+# Finally, we check to see what version of CALDB (if any) is installed - for the Chandra mission
+# Here we check to see whether CALDB is installed (along with all the necessary paths)
+CALDB_VERSION = None
+# This checks for an installation of Ciao
+CALDB_AVAIL = False
+
+if 'not installed' in split_out[5].lower():
+    warn("A Chandra CALDB installation cannot be identified on your system, and as such "
+         "Chandra data cannot be processed.")
```

Comment:
Same thing here as above with the stacklevel=2

Reply:
I didn't change the warning context for CALDB since, in theory, it's just for Chandara data processing, and we won't be using it for XGA.